Source/Core/Common/QoSSession.h

```diff
@@ -0,0 +1,72 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <enet/enet.h>
+
+namespace Common
+{
+class QoSSession
+{
+public:
+  // 1 0 1 1 1 0 0 0
+  // DSCP      ECN
+  constexpr static int ef_tos = 0b10111000;
```

Comment:
Our convention is to place lifetime specifiers before mutability ones (`static constexpr`).